dipy/reconst/ivim.py

```diff
@@ -509,6 +536,360 @@ def _leastsq(self, data, x0):
                 return x0
 
 
+class IvimModelVP(ReconstModel):
+
+    def __init__(self, gtab, fit_method='VarPro', *args, **kwargs):
+        r""" Initialize an IvimModelVP class.
+
+        The IVIM model assumes that biological tissue includes a volume
```

Comment:
Is there some way to avoid the duplication of the parts of the docstring that are identical between the two models?

Reply:
@skoudoro @Garyfallidis ?

Reply:
Personally, not aware of a clean way. Future work?